vita3k/emuenv/include/emuenv/state.h

```diff
@@ -137,6 +137,7 @@ struct EmuEnvState {
     IVector2 drawable_size = { 0, 0 };
     FVector2 viewport_pos = { 0, 0 };
     FVector2 viewport_size = { 0, 0 };
+    bool drop_inputs{};
```

Comment:
Put this in the ctrl state

Reply:
No relate ctrl state att all, it only relate emuenv

Reply:
I don't know, this variable is read only once in the function retrieve_ctrl_data in the file ctrl.cpp, it kind of seems like it is related to the controller state.

Reply:
no no at all look better, this variable is used and defined at the emulator level, in the interface file, the ctrl is only for reading, I mean this variable is assigned for vita3k itself, and not not related to the ctrl state

Reply:
Can you tell me what part of a variable meaning that we need to return nothing when the game asks for controller inputs is not related to the controller state?
Also the emuenv/state.h file is included by 400 files, it should contain nothing but pointers to other states.

Reply:
It include already very lot bool and very lot variable, again, this variable not relate ctrl att all, relate only emu env side for drop jnout when gui dialog is open